tools/jbrowse2/macros.xml

```diff
@@ -239,6 +240,7 @@
                 <option value="LinearPileupDisplay">LinearPileupDisplay - good for dense tracks at scale</option>
                 <option value="LinearArcDisplay">LinearArcDisplay</option>
             </param>
+            <param name="usebedscore" type="boolean" truevalue="true" falsevalue="false" checked="false" label="Use column 5 as a score for the display if bed format"/>            
```

Comment:
But the 5th value of a 6-column BED is the score. So why is this not the default? I'm not sure if I understand this parameter and its implications. If I take this off - set to false - which is now the default. Which column will it take instead?

Reply:
If off, nothing changes and it is just another jbrowse2 bed track so tests pass as usual.

There is no current way to use the bed score in JB2, other than turning that switch on.
That's why it was included for the specific use case.

Reply:
What about a help text in the like of "Use column 5 of your BED file as a score to drive different color schemas.""